src/test/java/microsoft/sql/DateTimeOffsetTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made

```

Comment:
driver tests are in the src/test/java/com/microsoft/sqlserver/jdbc folder.

also, this 1 test doesn't really need a separate file this can just be added to [DataTypeTests.java](https://github.com/microsoft/mssql-jdbc/blob/main/src/test/java/com/microsoft/sqlserver/jdbc/datatypes/DataTypesTest.java)